src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/dto/streamstats/StreamConfig.java

```diff
@@ -417,6 +423,125 @@ public String getDefaultStreamName() {
 		}
 	}
 
+	/**
+	 * This method is used to create command for stream control: create
+	 *
+	 * @return String CLI command
+	 */
+	public String contributeCommand(String command, String action) {
+		StringBuilder request = new StringBuilder();
+		request.append(command.concat(DecoderConstant.SPACE)
+				.concat(action));
+
+		Encapsulation encapsulationEnum = Encapsulation.getByApiName(getDefaultValueForNullData(this.encapsulation, DecoderConstant.EMPTY));
+		SRTMode srtModeEnum = SRTMode.getByName(getDefaultValueForNullData(this.srtMode, DecoderConstant.EMPTY));
+		SwitchOnOffControl aeEncryptedEnum = SwitchOnOffControl.getByName(getDefaultValueForNullData(this.srtSettings, DecoderConstant.EMPTY));
+		SwitchOnOffControl streamFlippingEnum = SwitchOnOffControl.getByName(getDefaultValueForNullData(this.streamFlipping, DecoderConstant.EMPTY));
+
+		if (!StringUtils.isNullOrEmpty(name)) {
+			request.append(" name=\"" + name + DecoderConstant.DOUBLE_QUOTATION);
```

Comment:
Convert request.append(" name=\"" + name + DecoderConstant.DOUBLE_QUOTATION); to
request.append(" name=\"").append(name).append(DecoderConstant.DOUBLE_QUOTATION);
Same for the rest of append.

Reply:
I fixed it